Cloud Application Manager/Billing/billing-menu.md

```diff
@@ -0,0 +1,49 @@
+{{{
+"title": "Billing menu",
+"date": "07-10-2018",
+"author": "Guillermo Sanchez",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+**In this article:**
+
+* Billing Dashboard
+* Billing Usage History
+___
+### Billing Dashboard
+When you click on **Billing** in the left side menu you will land into a dashboard page, where a bar graph with the billed amount for the latest months will be shown. You can also change the range of months being shown in the bar graph by selecting a different value from the to p-right dropdown on the graph.
+
+![billing-dashboard.png](../../images/cloud-application-manager/billing/billing-dashboard.png)
```

Comment:
change [billing-dashboard.png] to [Billing Dashboard]
Also image is not found...the paths are case-sensitive.

File '/home/travis/build/CenturyLinkCloud/PublicKB/images/cloud-application-manager/billing/billing-dashboard.png' not found. Is there a file extension? Are the file extension and path correct? (Referenced from '/home/travis/build/CenturyLinkCloud/PublicKB/Cloud Application Manager/Billing/billing-menu.md')

Either, rename the directory to `billing` or update the path to `Billing`